### ISSUE TITLE
import * bring in internal symbols when __all__ is not defined.

### ISSUE THREAD

**Issue by heejaechang** [microsoft/pyright#1530]
Note: if you are reporting a wrong signature of a function or a class in the standard library, then the typeshed tracker is better suited for this report: https://github.com/python/typeshed/issues.

**Describe the bug**
"import *" brings in internal symbols when __all__ in not defined.

**To Reproduce**
```python
import sqlite3

sqlite3. <= "os" shows up here.
``` 

**Expected behavior**
"os" not show up

**Screenshots or Code**
![image](https://user-images.githubusercontent.com/1333179/108926446-4cbeaf80-75f3-11eb-97fa-d2f3a4e6ec4e.png)

**VS Code extension or command-line**
vscode

**Comment by erictraut** [comment]
The logic in the binder for wildcards predated the introduction of the "externally hidden" flag, so the fix is to use this facility.

Here's the modification: https://github.com/microsoft/pyright/commit/041a6655481cd448a943cb8ef1da719f12e447bc

I've verified that the completion suggestion for sqlite3 is fixed with this change.

**Comment by jakebailey** [comment]
Seems to be failing in CI.

**Comment by erictraut** [comment]
This is addressed in Pyright 1.1.115, which was just published. It will also be in the next release of Pylance.
